app/models/host/discovered.rb

```diff
@@ -247,4 +247,14 @@ def create_notification
   def delete_notification
     ForemanDiscovery::UINotifications::DestroyHost.deliver!(self)
   end
+
+  def notification_recipients_ids
+    org_admins = Role.unscoped.find_by(
+      :name => Role::ORG_ADMIN,
+      :organization_id => organization_id
+    ).users.pluck(:id)
+    admins = User.unscoped.only_admin.except_hidden.
+      reorder('').distinct.pluck(:id)
+    (org_admins + admins).uniq
+  end
```

Comment:
Thanks, will this work also for cloned roles from ORG_ADMIN?

Reply:
I'm afraid search by `:name => Role::ORG_ADMIN` won't work, the workflow is that the user clones this template role, and rename it to e.g. "Org admin of Default org". They don't assign organizations to this template role directly.

Isn't there a specific permission that we should check? I guess all users in org that have permission to provision the discovered host and are assigned to this organization should be in the audience?